### ISSUE TITLE
I don't think any alt text that has a file extension as part of it, should pass

### ISSUE THREAD

**Issue by DavidMacDonald** [act-rules/act-rules.github.io#1246]
https://act-rules.github.io/rules/9eb3f6
Image filename is accessible name for image - Passed Example 2

I think the test should be deleted. If a blind person hears a file extension as part of an alt, they will not feel its dependable.

**Comment by patrickhlauke** [comment]
disagree in this case, as the link is for downloading a specific file. screen reader users will likely want to know the filename and extension to know the kind of file they're downloading (same way I, as a sighted user, would look at the browser's infobar bottom-left to see what the target of the link is)

**Comment by WilcoFiers** [comment]
Yeah, I agree with Patrick.  It's rarely the best solution, but I don't think having a filename is a definite failure of WCAG.

**Comment by Jym77** [comment]
@DavidMacDonald do you feel this needs more discussion, or should we just close the issue (according to the two answers)?

**Comment by WilcoFiers** [comment]
Given the lack of a response, I'm going to close the issue. My impression is that the majority of the CG thinks file extensions in a filename are not an automatic failure of SC 1.1.1, although it likely is an issue.
